mantaray/backend.py

```diff
@@ -293,6 +300,14 @@ def _handle_received_message(self, msg: _ReceivedAndParsedMessage) -> None:
             self.event_queue.put(UserQuit(msg.sender, reason or None))
             return
 
+        if msg.command == "KICK":
+            assert msg.sender is not None
+            kicker_nick = self.nick
```

Comment:
This seems wrong. The mantaray user isn't the only one who can kick.

Reply:
Changed from self.nick to msg.sender